qa/core-application-e2e-test-suite/utils/apiHelpers.ts

```diff
@@ -35,5 +37,46 @@ export async function authAPI(
       password: password,
     },
   });
-  await apiRequestContext.storageState({path: 'utils/.auth'});
+  // Save authentication session in the app-specific file
+  await apiRequestContext.storageState({path: authFilePath});
+}
+
+export async function pollAPI<T>(
```

Comment:
@esraagamal6 Did you try [`toPass`](https://playwright.dev/docs/test-assertions#expecttopass) as I recommended?

It's better to use default APIs so we have less things to maintain